docker-compose.yml

```diff
@@ -101,6 +101,17 @@ services:
     volumes:
       - data:/var/lib/postgresql/data
 
+  # Postgres for storing application data
+  user-postgres:
+    image: 'postgres:11.13'
+    env_file: ./userData/userData.env
```

Comment:
nit - can we use kebab or snake case on the userData directory & userData file to conform to the styling of the rest of this repo

Reply:
fixed!